web/app/js/components/LineGraph.jsx

```diff
@@ -133,6 +135,40 @@ export default class LineGraph extends React.Component {
     }
   }
 
+  flashLatestDataPoint() {
+    this.svg.select("circle").remove();
+    let circleData = {};
+
+    if (!_.isEmpty(this.props.data) && this.props.showFlash) {
+      circleData = this.props.data.slice(-1)[0];
+      this.svg.append("circle")
+        .attr("class", "flash")
+        .style("opacity", 0.6)
+        .attr("cx", () => {
+          if (circleData.timestamp) {
```

Comment:
I think we should check for timestamp and value before entering this loop and just return if it's not there, so we on't have to check here and in cy